unit_tests/ocean_waves/test_wave_theories.cpp

```diff
@@ -76,19 +76,44 @@ TEST_F(WaveTheoriesTest, StokesWaves)
 
 TEST_F(WaveTheoriesTest, StokesWaveLength)
 {
-
-    amrex::Real wave_height = 0.2;
-    amrex::Real wave_period = 2.2;
-    amrex::Real water_depth = 5.0;
-    int wave_order = 5;
+    // Values of wave_height, wave_period and water_depth taken from
+    // https://www.sciencedirect.com/science/article/pii/S0029801817306066
+    amrex::Real wave_height = 0.16;
+    amrex::Real wave_period = 1.6;
+    amrex::Real water_depth = 18.;
+    int wave_order = 2;
     constexpr amrex::Real g = 9.81;
     constexpr amrex::Real tol_lambda = 1e-10;
+    int iter_max =
+        -1; // To return computed wavelength from first guess of wavenumber k
```

Comment:
this is a minor thing, but could you put the comment above this line, allowing the variable spec to be its own line? I'll add it as a suggestion so it's an easy change.
```suggestion
    // To return computed wavelength from first guess of wavenumber k
    int iter_max = -1; 
```

Reply:
Sure thing! Thank you for pointing it out and for making it easy to change!